### PR TITLE
CART-175 added emergency Towing property

### PR DESCRIPTION
**Pull request by RomanFedytskyi**

### CODE REVIEW

File: dev/publicDev/admin.properties/style.scss [hunk 1]
```diff
@@ -216,6 +216,7 @@
   position: relative;
   // TODO: fix name
   .pa-img-wrap {
+    display: block;
```
Comment (dmitrij-borchuk) [inline dev/publicDev/admin.properties/style.scss]: Rmove it

File: dev/publicDev/admin.properties/tpl.jade [hunk 2]
```diff
@@ -11,6 +11,10 @@
           input(type="text", placeholder="Search property", ng-model="vm.searchPropertyValue")
           button.btn.btn-submit.search Search
         .wrap-admin-properties
+          .pa-item
+            a.pa-img-wrap(href="#/photos/emergency_towing")
```
Comment (dmitrij-borchuk) [inline dev/publicDev/admin.properties/tpl.jade]: Move "a" to one layer deepped

File: dev/publicDev/emergencyTowing/service.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+service.$inject = ['$http', 'TokenService'];
+function emergencyTowingService ($http, TokenService) {
+  return {
+    getEmergencyTowing: function() {
```
Comment (dmitrij-borchuk) [inline dev/publicDev/emergencyTowing/service.js]: Move to "dev\publicDev\photos\dataProvider.js" and rename "dev\publicDev\photos\dataProvider.js" service please (wrong name "propertiesService").

File: dev/publicDev/photos/ctrl.js [hunk 4]
```diff
@@ -6,37 +6,48 @@
...
     $routeParams,
     propertiesService,
-    AdminEmployeesService
+    AdminEmployeesService,
+    emergencyTowingService
   ) {
     var vm = this;
 
     vm.photos = [];
+    vm.emergencyTowing = [];
```
Comment (dmitrij-borchuk) [inline dev/publicDev/photos/ctrl.js:20]: Can we use "vm.photos" for emergency towing?

File: dev/publicDev/photos/tpl.jade [hunk 5]
```diff
@@ -7,5 +7,8 @@
           img(ng-src="{{photo}}")
           p {{photo.updated}}
           p Driver {{photo.ownerNumber}}
-        .message(ng-if="!vm.photos.length")
+        .photo(ng-repeat="photo in vm.emergencyTowing")
+          img(ng-src="{{photo}}")
+          p Property: {{photo.propertyName}}
+        .message(ng-if="!vm.photos.length && !vm.emergencyTowing.length")
```
Comment (dmitrij-borchuk) [inline dev/publicDev/photos/tpl.jade]: Can we use "vm.photos" for emergency towing?

**Review by dmitrij-borchuk — CHANGES_REQUESTED** [review]

File: dev/publicDev/photos/tpl.jade [hunk 6]
```diff
@@ -7,5 +7,8 @@
           img(ng-src="{{photo}}")
           p {{photo.updated}}
           p Driver {{photo.ownerNumber}}
-        .message(ng-if="!vm.photos.length")
+        .photo(ng-repeat="photo in vm.emergencyTowing")
+          img(ng-src="{{photo}}")
```
Comment (dmitrij-borchuk) [inline dev/publicDev/photos/tpl.jade]: "photo" is not image url

**Review by dmitrij-borchuk — CHANGES_REQUESTED** [review]

File: dev/publicDev/admin.properties/style.scss (hunk 1 above)
Comment (RomanFedytskyi) [reply dev/publicDev/admin.properties/style.scss]: it should be because its tag 'a'

Comment (dmitrij-borchuk) [reply dev/publicDev/admin.properties/style.scss]: Move "a" to one layer deepped

File: dev/publicDev/photos/tpl.jade (hunk 5 above)
Comment (RomanFedytskyi) [reply dev/publicDev/photos/tpl.jade]: vm.photos it`s photo for one property and vm.emergencyTowing its for all property

**Review by dmitrij-borchuk — APPROVED** [review]

**Pull request merged**